includes/functions/llms.functions.certificate.php

```diff
@@ -110,6 +110,66 @@ function llms_get_certificate_fonts() {
 
 	$serif = '"Iowan Old Style", "Apple Garamond", Baskerville, "Times New Roman", "Droid Serif", Times, "Source Serif Pro", serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"';
 
+	$fonts = array(
```

Comment:
this change doesn't really do what we'd like it to do. It does provide devs with a way to opt back into using google fonts, kind of, but not really in the way that I envisioned it (it results in a more convoluted switch rather than using a `__return_true` callback on `add_filter()`

Please refer back to my original suggestion on using a ternary on the `href` property of each font.

Let me know if you have any questions on my desired implementation